.setup/testing/autograder.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
```

Comment:
Can we combine `.setup/testing/*` and `.setup/travis/*` files? Having them separate means we risk updating one and not the other until Travis is completely killed off which I'd like to avoid as it's added mental load, and I do not believe the files do that much that's different between the two. At worst, can probably gatekeep based on the `${TRAVIS}` env variable.

Reply:
I'd like to keep them separate so when travis gets removed I can just delete the entire directory, it would also save me time in debugging trying to merge them back together which I don't think will really be worth it because the last two tests are integration and the git tests which I expect to be moved over pretty soon.

But yeah they are pretty similar maybe I can get away with defining 'travis' only blocks that can easily be removed once we're done with it.